doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -709,48 +716,84 @@ Response:
 
 + Response 204
 
-## Context Entity Attributes [/v2/entities/{entityId}/attrs]
+## Entity Attribute [/v2/entities/{entityId}/attrs/{attrName}]
 
-### List all attributes [GET /v2/entities/{entityId}/attrs]
```

Comment:
I understand that at the end "GET /v2/entities/{entityId}/attrs" is removed from the API to keep things lean, given that "GET /v2/entity/{entityId}" is basically the same.